.evergreen/config/test/_template-ocsp.yml

```
@@ -0,0 +1,389 @@
+tasks:
+  - name: "ocsp-test_1-rsa-delegate-%mongodbVersion%"
+    tags: ["ocsp", "%mongodbVersion%"]
+    commands:
+      - func: "run OCSP responder"
+        vars:
+          TEST_COLUMN: "TEST_1"
+          CERT_TYPE: "rsa"
+          USE_DELEGATE: "on"
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          MONGODB_VERSION: "%mongodbVersion%"
+          TOPOLOGY: "server"
+          ORCHESTRATION_FILE: "rsa-basic-tls-ocsp-mustStaple.json"
+      - func: "run tests"
+        vars:
+          TESTS: "tests/ocsp-success.phpt"
+          APPEND_URI: "/?tls=true&tlsCAFile=${DRIVERS_TOOLS}/.evergreen/ocsp/rsa/ca.pem"
+  - name: "ocsp-test_1-ecdsa-delegate-%mongodbVersion%"
```

```suggestion

  - name: "ocsp-test_1-ecdsa-delegate-%mongodbVersion%"
```

Likewise, newlines would help with readability.